machine_management/creating-infrastructure-machinesets.adoc

```diff
@@ -44,8 +44,18 @@ by adding SpotMarketOptions to the MachineSet YAML file.
 
 include::modules/machineset-yaml-azure.adoc[leveloffset=+3]
 
+Azure MachineSets support non-guaranteed xref:../machine_management/creating_machinesets/creating-machineset-azure.html#machineset-non-guaranteed-instance_creating-machineset-azure[Spot VMs],
```

Comment:
This xref should refer to the .adoc file not the .html file. For example:

xref:../machine_management/creating_machinesets/creating-machineset-azure.adoc#machineset-non-guaranteed-instance_creating-machineset-azure[Spot VMs]